doc/tutorials/rt_industry.rst

```diff
@@ -0,0 +1,456 @@
+   |                      |                  |                      | - NVMe: 256G Intel Corporation SSD Pro 7600p/760p/E 6100p |
+   +----------------------+------------------+----------------------+-----------------------------------------------------------+
+
+Software Setup
+==============
+
+.. table:: Software Setup
+   :widths: auto
+   :name: Software Setup
+
+   +---------------------+------------------------+------------------------------------------+------------------------------------------------------------------------------------------------+
+   | Clear Linux Version | ACRN tag (hypervisor)  | Description                              | Download Link                                                                                  |
+   +=====================+========================+==========================================+================================================================================================+
+   | 29750               |                        | **Host machine:**                        | `link <https://download.clearlinux.org/releases/29750/clear/clear-29750-live-server.iso.xz>`__ |
+   |                     |                        | build ACRN hypervisor, tools and kernels |                                                                                                |
+   +---------------------+------------------------+------------------------------------------+------------------------------------------------------------------------------------------------+
+   | 30690               | acrn-2019w31.5-140000p | **KBL NUC:**                             | `link <https://download.clearlinux.org/releases/30690/clear/clear-30690-live-server.iso.xz>`__ |
+   |                     |                        | set up SOS or Native OS                  |                                                                                                |
+   +---------------------+------------------------+------------------------------------------+------------------------------------------------------------------------------------------------+
+
+Install build tools and dependencies
+====================================
+On the Clear Linux build machine, install the necessary tools in order to build ACRN
+hypervisor, Service VM and Preempt-RT kernels::
+
+  $ sudo swupd bundle-add os-clr-on-clr os-core-dev python3-basic
+  $ pip3 install --user kconfiglib
+
+How to rebuild ACRN Hypervisor for industry scenario
+****************************************************
+In this section, we firstly follow :ref:`quick-setup-guide` to set up the Service VM.
+After setup, we need to rebuild acrn-hypervisor by enabling industry scenario and
+replace ``acrn.efi``.
+
+Set up the Service VM on KBL SATA Disk
+======================================
+#. Refer to the :ref:`quick-setup-guide` to boot Service VM:
+
+   .. code-block:: console
+      
+      $ export https_proxy=https://myproxy.mycompany.com:port
```

Comment:
I prefer that we remove all references to proxy setup information, I see this as out of scope for our documentation.

Reply:
It's removed in new release.